crates/ruff_benchmark/src/lib.rs

```diff
@@ -68,6 +69,28 @@ pub struct TestFile {
     code: String,
 }
 
+static TARGET_DIR: once_cell::sync::Lazy<PathBuf> = once_cell::sync::Lazy::new(|| {
```

Comment:
This fixes an issue where the benchmarks created a target folder in the `ruff_benchmark` directory instead of re-using Cargo's `target` directory (copied from criterion)